### ISSUE TITLE
Handling negative timesteps

### ISSUE THREAD

**Issue by paulrossmanith** [Mayitzin/ahrs#30]
Hello there!

First of all, thank you for providing such an amazing toolbox, I really appreciate the time and effort spent! It helped me out tremendously!
However, when using e.g.  `madgwick.Dt` to set the filters sampling time, negative values are also accepted. It should at least spew a warning to save some time debugging. :) 

Cheers!

**Comment by Mayitzin** [comment]
Good idea. Although the prevention of error from the user input won't be high up in the priorities. Perhaps for the next release I can help with that.
